coursemapper-kg/preprocess/Dockerfile

```
@@ -1,23 +1,35 @@
-		lbzip2 \
-		postgresql-client-16 \
-		rclone \
-		&& rm -rf /var/lib/apt/lists/*
-RUN pip install --no-color pipenv==2023.12.1
+# Install Python packages
+COPY Pipfile Pipfile.lock ./
+RUN pip install --no-color --disable-pip-version-check --no-cache-dir pipenv==2023.12.1
 RUN pipenv install --deploy
 
+# Remove build dependencies
+RUN	apt-get remove -yq $BUILD_DEPS
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)